src/Equinox/Decider.fs

```diff
@@ -244,9 +244,9 @@ and internal LoadPolicy() =
     static member Fetch<'state, 'event>(x: LoadOption<'state> option)
         : IStream<'event, 'state> -> CancellationToken -> Task<struct (StreamToken * 'state)> =
         match x with
-        | None | Some RequireLoad ->                 fun stream ct ->   stream.Load(allowStale = false, requireLeader = false, ct = ct)
-        | Some RequireLeader ->                      fun stream ct ->   stream.Load(allowStale = false, requireLeader = true,  ct = ct)
-        | Some AllowStale ->                         fun stream ct ->   stream.Load(allowStale = true,  requireLeader = false, ct = ct)
+        | None | Some RequireLoad ->                 fun stream ct ->   stream.Load(maxStaleness = TimeSpan.Zero,     requireLeader = false, ct = ct)
+        | Some RequireLeader ->                      fun stream ct ->   stream.Load(maxStaleness = TimeSpan.Zero,     requireLeader = true,  ct = ct)
+        | Some AllowStale ->                         fun stream ct ->   stream.Load(maxStaleness = TimeSpan.MaxValue, requireLeader = false, ct = ct)
```

Comment:
Nit: in theory a system could be running beyond the heat death of the universe and get a stale value in this branch 🤡

Reply:
not unrelated, I was pondering whether the etag in DynamoStore should just become an int64 update count - would save lots of allocations to boot

Reply:
That makes total sense to me

Reply:
>That makes total sense to me
Running? Wanting to break the format for DynamoStore and/or write tests to validate that an impl has backcompatiblity? :D
If only there were phalanxes of contributors ready to pounce on up-for-grabs issues for the kudos IRL :D